lib/services/bundler/bundler-compiler-service.ts

```
@@ -1078,15 +1094,31 @@ export class BundlerCompilerService
 			case "rspack":
 				return true;
 			default:
-				const packageJSONPath = resolvePackageJSONPath(WEBPACK_PLUGIN_NAME, {
-					paths: [projectData.projectDir],
-				});
+				const packageJSONPath = resolvePackageJSONPath(
+					this.getBundlerPackageName(),
+					{
+						paths: [projectData.projectDir],
+					},
+				);
```

Biome (2.5.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)